aiohttp_wsgi/__main__.py

```diff
@@ -331,4 +332,6 @@ def main():  # pragma: no cover
             pass
 
 
-__doc__ = __doc__.format(**HELP, help=textwrap.indent(parser.format_help(), "    "))
+__doc_args = HELP.copy()
+__doc_args['help'] = textwrap.indent(parser.format_help(), "    ")
+__doc__ = __doc__.format(**__doc_args)
```

Comment:
Again here, I think we just need to put **HELP last, without a trailing comma to support Python 3.4